portfolio/src/main/webapp/index.html

```diff
@@ -5,6 +5,7 @@
     <title>Gábor Fuisz - 2020 STEP Internship Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="CommentSection.js"></script>
```

Comment:
rename this file to use either - or _ as a separator instead of CamelCase. E.g. `comment_section.js`

https://google.github.io/styleguide/jsguide.html#file-name

Reply:
I'm afraid it would cause some git problems, I will rename it in a later commit

Reply:
I understand that. Do that in a later commit and treat it as if it was the same to see what the potential issues would be. Note that git does not support marking a file as moved to another location but tries to automatically infer that when it sees a file being deleted and another one that is very similar being added. Therefore whenever you rename or move existing files, make sure to do so in an individual commit which only changes exactly what is needed for the move.